test/src/e2e_vm_tests/test_programs/cei_warn/src/main.sw

```diff
@@ -0,0 +1,21 @@
+contract;
+
+use std::storage::store;
+
+// TODO when the test harness can expect warnings, _this should warn about CEI_!
```

Comment:
Do we have an issue for warning awareness in the test harness? Could be useful to link to one here maybe.

Reply:
I don't think the current test harness care about warnings (the standard library is full of them :joy:). But we should be able to configure which errors/warnings the compiler checks, so that downstream users of `forc` (including the test suite) can disable some.

Reply:
This _loosely_ tracks test harness desires, we could use this issue as a bikeshedding place:

#723